.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
What is this? It seems specific to your setup and should probably be in .gitignore and not added to the project.